### ISSUE TITLE
Screen readers can't access alt text on posts with multiple images 

### ISSUE THREAD

**Issue by str4d** [bluesky-social/social-app#947]
**Describe the bug**

When a post has a single image, screen readers can find the alt text and read it out.

When a post has multiple images, the alt text for each image cannot be found by screen readers:
- [This user reports](https://bsky.app/profile/paradoxikat.bsky.social/post/3jzl3bhv6he26) that all four images read "view image" as the text for their alt attribute.
- [This user reports](https://bsky.app/profile/oedipusnj.bsky.social/post/3jzlbucn2gf2j) hearing the word "ALT" repeated four times immediately after the post's text was read out.

**To Reproduce**

- Open a post with multiple images ([this post](https://bsky.app/profile/chadloder.bsky.social/post/3jziw3suo242a) is the one that led to the discovery of the issue).
- Attempt to navigate the post with a screen reader to discover the alt text of each image.

**Expected behavior**

The images within the post should be individual focusable items, so the screen reader can tab between each of them to read out the alt text of a single image at a time.

**Details**

 - Platform: [Reported as affecting both desktop Chrome and desktop Firefox](https://bsky.app/profile/paradoxikat.bsky.social/post/3jzlabxviuo26), unknown OS.
 - Platform version: Unknown, presumably latest.
 - App version: Presumably latest (1.37.0), these reports are from the last day.

**Additional context**

I opened this issue based on the reports on Bluesky; I have not reproduced the issue myself.

**Comment by pfrazee** [comment]
Should hopefully be fixed by https://github.com/bluesky-social/social-app/pull/949
